### ISSUE TITLE
Cannot see character under cusor with the Obsidian color scheme

### ISSUE THREAD

**Issue by odrevet** [wez/wezterm#701]
## Describe the bug

Cannot see character under cursor with the Obsidian color scheme

## Environment 

 - OS: Linux Wayland

## Configuration

`color_scheme = "Obsidian",`

I like this theme, could the cusror color be fixed ? Meanwhile I am using a sticky cursor, but is it possible to override cursor theme color in configuration ? cursor_bg and cursor_fg does not seems to change the cursor color then a color scheme is used

**Comment by wez** [comment]
The color schemes are maintained by the upstream project: https://github.com/mbadolato/iTerm2-Color-Schemes
If you want to adjust it in the upstream then I'd suggest proposing the change over there.

`color_scheme` is defined as overriding the `colors` section of your config, so probably your easiest option is to define your own local version of Obsidian in your config:

https://wezfurlong.org/wezterm/config/appearance.html#defining-a-color-scheme-in-your-weztermlua has more information on this, and you can find the definition of Obsidian over here: https://github.com/wez/wezterm/blob/main/assets/colors/Obsidian.toml

In other words:

```lua
return {
  color_scheme = "My Obsidian",
  color_schemes = {
    ["My Obsidian"] = {
      foreground = "#cdcdcd",
      background = "#283033",
      cursor_bg = "#c0cad0",
      cursor_border = "#c0cad0",
      cursor_fg = "black", -- fixed!
      selection_bg = "#3e4c4f",
      selection_fg = "#dfe1e2",
      ansi = {"#000000","#a60001","#00bb00","#fecd22","#3a9bdb","#bb00bb","#00bbbb","#bbbbbb"},
      brights = {"#555555","#ff0003","#93c863","#fef874","#a1d7ff","#ff55ff","#55ffff","#ffffff"},
   },
 },
}
```

**Comment by odrevet** [comment]
thanks
